src/sentry/replays/usecases/replay_counts.py

```diff
@@ -73,15 +77,19 @@ def _get_replay_id_mappings(
 
     for row in discover_results["data"]:
         for replay_id in row["group_uniq_array_100_replayId"]:
-            replay_id_to_issue_map[replay_id].append(row[select_column])
+            # When no replay exists these strings are provided in their empty
+            # state rather than null. This can cause downstream problems so
+            # we filter them out.
+            if len(replay_id) == 32:
```

Comment:
should this check be against string null or empty instead of checking for the key's current length?

Reply:
Yes.  I was thinking by doing this I could guard against more forms of malformed input but I think I'd rather 500 than cover them up.  I'll explicitly target `""`.